docs/installing/preparing_edge_cluster.md

```diff
@@ -31,7 +31,7 @@ Install one of these edge clusters and prepare it for the {{site.data.keyword.ed
 
 1. Install OCP by following the installation instructions in the [{{site.data.keyword.open_shift_cp}} Documentation ](https://docs.openshift.com/container-platform/4.5/welcome/index.html){:target="_blank"}{: .externalLink}. ({{site.data.keyword.ieam}} only supports OCP on x86_64 platforms.)
 
-2. Install the Kubernetes CLI (**kubectl**), Openshift client CLI (**oc**) and Docker on the admin host where you administer your OCP edge cluster. This is the same host where you run the agent installation script. For more information, see [Installing cloudctl, kubectl, and oc](../cli/cloudctl_oc_cli.md).
+2. Install the Kubernetes CLI (**kubectl**), Openshift client CLI (**oc**) and Docker on the admin host where you administer your OCP edge cluster. This is the same host where you run the agent installation script. For more information, see [Installing cloudctl, kubectl, and oc](../nl/es/cli/cloudctl_oc_cli.md).
```

Comment:
The cloudctl_oc_cli document was contributed by IBM and does not appear to apply to Open Horizon.  That's why the file does not exist in US English for Open Horizon.  We should unlink this file unless and until we create an Open Horizon-specific version of the page.

Reply:
so do I need to add an empty link or remove that para in order to unlink it?

Reply:
How about removing the "For more information" sentence for now.  Can you open a separate issue identifying this page (docs/installing/preparing_edge_cluster.md) as a candidate for re-writing?

Reply:
sure, will do that :)

Reply:
I am opening an issue for the re-writing the page  (docs/installing/preparing_edge_cluster.md) so can you please provide me more details about it, like what changes to be specific we need?